apps/helpers.py

```diff
@@ -2,7 +2,8 @@
 from enum import Enum
 from typing import Optional
 
-from django.core.exceptions import ObjectDoesNotExist
+import regex as re
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
```

Comment:
Isort was not applied here

Reply:
https://scilifelab.slack.com/archives/C02L5Q7HWR1/p1732872632171369?thread_ts=1732623632.294409&cid=C02L5Q7HWR1